src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class IntakeSubsystem extends SubsystemBase {
+
+    private final TalonFX intakeMotor;
+
+    private final DigitalInput rightBeamBreak;
+    private final DigitalInput leftBeamBreak;
+
+    public IntakeSubsystem() {
+        intakeMotor = new TalonFX(0); 
+        intakeMotor.setNeutralMode(NeutralModeValue.Brake);
+        //intakeMotor.setInverted(true);
+        rightBeamBreak = new DigitalInput(0);
+        leftBeamBreak = new DigitalInput(0);
+    }
+
+    public void setMotorSpeed(double speed) {
```

Comment:
I think exposing a speed method like this is pretty reasonable, since there's no real benefit in, say, using RPM.

Is a positive speed intake, or eject? You probably don't want people to have to know. Consider `setIntakeSpeed`, `setEjectSpeed`, and perhaps `stop`. Both sets can take a positive value 0-1, and now you can bind them to different triggers.